optin/reporting/test/controllers/completedController.spec.js

```diff
@@ -63,100 +70,93 @@ describe('completed controller', () => {
 			expect(result.symbolBalance).to.be.eql(mockSymbolSources.map(symbolInfo => symbolInfo.balance));
 		};
 
+		const runBasicQueryTests = async overWriteParams => {
+			// Arrange:
+			req = TestUtils.mockRequest({
+				...parameter,
+				...overWriteParams
+			});
+
+			getCompletedPaginationStub.returns(Promise.resolve([]));
+
+			// Act:
+			await CompletedController.getCompleted(req, res);
+
+			// Assert:
+			const result = res.json.getCall(0).firstArg;
+			expect(result).not.to.haveOwnProperty('error');
```

Comment:
as you are only verifying there is no error, this is a weak test. it's probably ok for this, but you should use stricter asserts in future.